src/content/en/fundamentals/accessibility/00-intro/index.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: shared/narrow-pages-list
+title: "Get Started"
+description: "Learn the basics of accessibility, what it means and why it's important."
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 1
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+Understanding the accessibility issue, its scope, and its impact can make you a better web developer.
```

Comment:
Would rephrase as:

"Understanding accessibility, its scope, and its impact can make you a better web developer"